changelogs/fragments/65372-misc-context-manager.yml

```
@@ -0,0 +1,6 @@
+bugfixes:
+  - spacewalk inventory - improve file handling by using a context manager.
+  - intersight remote management - improve file handling by using a context manager.
```

There are only two intersight modules, so it's best to list them up:
```suggestion
  - intersight_rest_api, intersight_info - improve file handling by using a context manager.
```